_posts/2018-03-23-cache-breaking-for-faster-web-application-updates.md

```diff
@@ -10,26 +10,28 @@ tags: []
 ---
 
 ### When Production Is Broken, It Needs to Be Fixed Now!
-Recently, a 3rd party changed the URL to one of their map services that we use in several of our applications. This came as a bit of a surprise to us so we quickly updated our applications and deployed new versions. Everything seemed great and we continued on our way. However, a full two days later, one of our clients came to me reporting that he still had users that did not have the updated applications in their browsers. The need to teach clients how to clear their browser caches feels unprofessional and it is unnecessary for them and me. So I started doing some digging on how this problem could be solved.
+Recently, the URL changed for one of the map services we use in several of our applications. This came as a bit of a surprise to us, so we quickly updated our applications and deployed new versions. Everything seemed great and we continued on our way. However, two days later one of our clients reported that some of his users still did not have the updated applications in their browsers. The need to teach clients how to clear their browser caches felt unprofessional and unnecessary for them and me. So I started doing some digging on how this problem could be solved.
```

Comment:
Regarding _"The need to teach clients how to clear their browser caches felt unprofessional and unnecessary for them and me."_: I'm not quite sure what we mean here.

Reply:
Without the cache busting technique he describes, you need to tell/teach a user how to clear their browser cache to get the latest version of the resources from the server. Otherwise they are still shown the old resources.